From 8da9fc0e489b940b5955e8fa5e1ceb0981e634fe Mon Sep 17 00:00:00 2001
From: dyphire <qimoge@gmail.com>
Date: Mon, 7 Jul 2025 02:25:47 +0800
Subject: [PATCH] apply patch

---
 packages/mpv.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 49df758..ff5d0bf 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -26,6 +26,7 @@ ExternalProject_Add(mpv
     GIT_REPOSITORY https://github.com/mpv-player/mpv.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    PATCH_COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/16583.patch | git am --3way --whitespace=fix
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 meson setup <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.47.0.windows.1

