From ab6c2698c5a64685049aea001e215a24d4539167 Mon Sep 17 00:00:00 2001
From: dyphire <qimoge@gmail.com>
Date: Thu, 5 Jun 2025 16:00:37 +0800
Subject: [PATCH] test

---
 packages/mpv.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 49df758..089bd38 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -26,6 +26,8 @@ ExternalProject_Add(mpv
     GIT_REPOSITORY https://github.com/mpv-player/mpv.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    PATCH_COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/15936.patch | git am --3way --whitespace=fix
+          COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/16417.patch | git am --3way --whitespace=fix
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 meson setup <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.47.0.windows.1

