From 516dec7265101d815c3170c21a5e03d1a26f4527 Mon Sep 17 00:00:00 2001
From: dyphire <qimoge@gmail.com>
Date: Fri, 28 Feb 2025 13:05:03 +0800
Subject: [PATCH] mpv: apply patch

---
 packages/mpv.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 49df758..8d9a1a6 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -26,6 +26,8 @@ ExternalProject_Add(mpv
     GIT_REPOSITORY https://github.com/mpv-player/mpv.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    PATCH_COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/15936.patch | git am --3way --whitespace=fix
+          COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/16125.patch | git am --3way --whitespace=fix
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 meson setup <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.47.0.windows.1

