From 84a38c39e40c0df65b6382cdac1e54dc32a9f217 Mon Sep 17 00:00:00 2001
From: dyphire <qimoge@gmail.com>
Date: Thu, 15 May 2025 14:15:39 +0800
Subject: [PATCH] mpv: apply patch

---
 packages/mpv.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/mpv.cmake b/packages/mpv.cmake
index 49df758..285d9d5 100644
--- a/packages/mpv.cmake
+++ b/packages/mpv.cmake
@@ -26,6 +26,7 @@ ExternalProject_Add(mpv
     GIT_REPOSITORY https://github.com/mpv-player/mpv.git
     SOURCE_DIR ${SOURCE_LOCATION}
     GIT_CLONE_FLAGS "--filter=tree:0"
+    PATCH_COMMAND ${EXEC} curl -sL https://github.com/mpv-player/mpv/pull/15936.patch | git am --3way --whitespace=fix
     UPDATE_COMMAND ""
     CONFIGURE_COMMAND ${EXEC} CONF=1 meson setup <BINARY_DIR> <SOURCE_DIR>
         --prefix=${MINGW_INSTALL_PREFIX}
-- 
2.47.0.windows.1